docs/meshstack.aws.landing-zone-tutorial.md

```diff
@@ -0,0 +1,266 @@
+            - s3:*
+            - sns:*
+            Resource: '*'
+```
+
+### OrganizationAccountAccessRole
+
+```yaml
+AWSTemplateFormatVersion: 2010-09-09
+Description: Configure the OrganizationAccountAccessRole
+
+Resources:
+  OrganizationAccountAccessRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: OrganizationAccountAccessRole
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              AWS:
+                - arn:aws:iam::${RootAccountId}:root
+            Action:
+              - sts:AssumeRole
+      Path: /
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/AdministratorAccess
+  MeshfedServiceAccessRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: OrganizationAccountAccessRole # tbd check for collisions with existing roles
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              AWS:
+                - arn:aws:iam::${RootAccountId}:meshfed-service
+            Action:
+              - sts:AssumeRole
+      Path: /
+      PolicyDocument:
+        Statement:
+          - Sid: IamReadOnly
+            Effect: Allow
+            Action:
+                - iam:GetRole
+                - iam:ListSAMProviders
+                - iam:ListRoles
+                - iam:GetSAMLProvider
+            Resource: '*' # tbd this may need to be more limited, but needs access on the LZ StackSet
+          - Sid: IamUpdateSAMLProvider
+            Effect: Allow
+            Action:
+                - iam:UpdateSAMProvider
+            Resource: arn:aws:iam::*:saml-provider/meshstack-saml-idp
+```
+
+### AWSCloudFormationStackSetExecutionRole
+
+```yaml
+AWSTemplateFormatVersion: 2010-09-09
+Description: Configure the AWSCloudFormationStackSetExecutionRole to enable use of your account as a target account in AWS CloudFormation StackSets. For simplicity, this role receives full AdministratorAccess on the target account.
+
+Resources:
+  ExecutionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: AWSCloudFormationStackSetExecutionRole
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              AWS:
+                - arn:aws:iam::${AutomationAccountId}:role/LZCFNStackSetAdministrationRole
+            Action:
+              - sts:AssumeRole
+      Path: /
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/AdministratorAccess
+```
+
+1. As a last step setup an AWS Landing Zone in meshStack. You need the URL of the above defined Permission Setup Template, the actual Stack Set template you wish to apply, the Account ID of the admin account which will contain the stack sets and the StackSet Admin Region (the region in which the StackSet in the Admin account is placed) as well as the StackInstance Deploy Region.
```

Comment:
Lastly, setup an AWS Landing Zone in meshStack. You will need the URL of the above defined Permission Setup Template, the actual Stack Set template you wish to apply, the Account ID of the admin account which will contain the stack sets, and the StackSet Admin Region (the region in which the StackSet in the Admin account is placed) as well as the StackInstance Deploy Region.